src/coreclr/src/System.Private.CoreLib/src/Internal/Runtime/InteropServices/ComponentActivator.cs

```diff
@@ -153,5 +128,61 @@ private static IsolatedComponentLoadContext GetIsolatedComponentLoadContext(stri
 
             return alc;
         }
+
+        private static AssemblyLoadContext InternalLoadAssembly(string assemblyPath)
```

Comment:
What's the point of wrapping GetIsolatedComponentLoadContext by another method?

Reply:
https://github.com/dotnet/runtime/pull/37696#discussion_r438061553. It's easy to remove `InternalLoadAssembly` until we actually need it if we don't want it today.

Reply:
I would also prefer to not have this method (if we ever introduce a native hosting way to create an ALC, it would have to take the parameter as IntPtr anyway, so I don't really see the usefulness of this wrapper).